pkg/middleware/headers.go

```diff
@@ -12,3 +16,39 @@ func HandleNoCacheHeader() macaron.Handler {
 		ctx.SkipCache = ctx.Req.Header.Get(HeaderNameNoBackendCache) == "true"
 	}
 }
+
+func AddSeceureResponseHeaders() macaron.Handler {
+	return func(res http.ResponseWriter, req *http.Request, c *macaron.Context) {
+
+		secureMiddleware := secure.New(createSecureOptions())
+
+		nonce, _ := secureMiddleware.ProcessAndReturnNonce(res, req)
```

Comment:
Why is the error ignored here? I would handle it, although I'm not sure off the top of my head how to in these handlers.

Reply:
I guess if an error occurred the function should just "return", like any other middleware behaviour that doesn't have reqContext.